features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/SessionReplayFeature.kt

```diff
@@ -90,9 +93,20 @@ internal class SessionReplayFeature(
 
         val resourcesFeature = registerResourceFeature(sdkCore)
 
+        val resourcesDataStoreManager = ResourcesDataStoreManager(
+            featureSdkCore = sdkCore,
+            resourceHashesSerializer = ResourceHashesEntrySerializer(),
+            resourcesHashesDeserializer = ResourceHashesEntryDeserializer(internalLogger = sdkCore.internalLogger)
```

Comment:
for the naming consistency

```suggestion
            resourceHashesDeserializer = ResourceHashesEntryDeserializer(internalLogger = sdkCore.internalLogger)
```